### PR TITLE
docs: update documentation

### PR DESCRIPTION
**Pull request by git-anurag-hub**
## Description

General documentation update!

## Checklist

- [x] My code follows the [contributing guidelines][contributing] of this project.
- [x] I am aware that all my commits will be squashed into a single commit, using the PR title as the commit message.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code in hard-to-understand areas.
- [x] I have updated the user-facing documentation to describe any new or changed behavior.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have not reduced the existing code coverage.

## Comments

[contributing]: CONTRIBUTING.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # ELIXIR Cloud Components
 
-Elixir Cloud Components is lightweight, fast, and reusable web-components developed by Elixir Cloud and AAI Community for managing and accessing various services.
+Elixir Cloud Components (ecc) is lightweight, fast, and reusable web-components developed by Elixir Cloud and AAI Community for managing and accessing various services.
```
Comment (uniqueg) [inline README.md:3]: ELIXIR Cloud Components *(ECC) are* lightweight, fast and reusable Web Components for operationalizing various [ELIXIR Cloud & AAI](https://elixir-cloud.dcc.sib.swiss/) and [GA4GH](https://ga4gh.org/) [Cloud](https://www.ga4gh.org/work_stream/cloud/) services.

**Review by uniqueg — CHANGES_REQUESTED** [review]
Small change suggested. Also, what's up with all those linter warnings?

**Comment by uniqueg** [comment]
Yes, I had seen that. But should we maybe create an issue to get rid of those warnings (in another PR)?

**Review by uniqueg — APPROVED** [review]

**Comment by git-anurag-hub** [comment]
> Yes, I had seen that. But should we maybe create an issue to get rid of those warnings (in another PR)?

Sure will look into it!

**Pull request merged**